pkg/config/descriptors/descriptors.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 the Istio Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package descriptors
+
+import (
+	dpb "istio.io/api/mixer/v1/config/descriptor"
+	pb "istio.io/mixer/pkg/config/proto"
+)
+
+// Finder describes anything that can provide a view into the config's descriptors by name and type.
+type Finder interface {
+	// GetLog retrieves the log descriptor named `name`
+	GetLog(name string) (*dpb.LogEntryDescriptor, bool)
```

Comment:
should these be named `Find*` to match `Finder` interface?  or should we name this interface `Getter` ?